notebooks/en/enterprise_dedicated_endpoints.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>Readability rewrite:</p><blockquote>You can use InferenceClient to easily send requests to your endpoint. It's a convenient utility available in the <code>huggingface_hub</code> Python library that allows you to easily make calls to both <a href="https://huggingface.co/docs/inference-endpoints/index" target="_blank">Dedicated Inference Endpoints</a> and the <a href="https://huggingface.co/docs/api-inference/index" target="_blank">Serverless Inference API</a>.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>